inc/auth.class.php

```diff
@@ -848,7 +848,7 @@ function login($login_name, $login_password, $noauto = false, $remember_me = fal
                   if (Toolbox::canUseLdap()) {
                      AuthLDAP::tryLdapAuth($this, $login_name, $login_password,
                                              $this->user->fields["auths_id"]);
-                     if (!$this->auth_succeded && !$this->user_found) {
+                     if (!$this->auth_succeded && $this->user_found) {
```

Comment:
This is not a typo. User should be deleted if not found in LDAP and has `'authtype' => $this::LDAP`.

Reply:
Ok, then we can set `$user_deleted_ldap` only if the the search on user + type ldap was successful.
Users with mail login won't be found by this and can be handled correctly by the next block.